### PR TITLE
Bookmark acts safe

### PR DESCRIPTION
**Pull request by agnesbr**

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -15,9 +15,13 @@ export default class App extends Component {
       reformatedFestivals[festIndex].festStartDate = new Date(
         festival.festStartDate
       )
-      reformatedFestivals[festIndex].festEndDate = new Date(
-        festival.festEndDate
-      )
+
+      if (reformatedFestivals[festIndex].festEndDate !== '') {
```
Comment (patryem) [inline src/components/App.js:19]: use `!= null` to check for falsy statements

File: src/components/App.js [hunk 2]
```diff
@@ -27,14 +31,15 @@ export default class App extends Component {
         ].actEndDate = new Date(act.actEndDate)
       })
     })
+    console.log(reformatedFestivals)
```
Comment (patryem) [inline src/components/App.js:34]: better take this out

File: src/components/App.js [hunk 3]
```diff
@@ -27,14 +31,15 @@ export default class App extends Component {
...
   }
 
   render() {
     this.reformatData()
-    console.log(this.state.festivals)
+
     return (
       <Router>
-        <React.Fragment>
```
Comment (patryem) [inline src/components/App.js:37]: React.Fragment id div is not needed

File: src/components/Fest.js [hunk 4]
```diff
@@ -23,8 +23,8 @@ export default class Fest extends Component {
   static propTypes = {
     festId: PropTypes.number.isRequired,
     festName: PropTypes.string.isRequired,
-    festStartDate: PropTypes.date,
-    festEndDate: PropTypes.date,
+    festStartDate: PropTypes.instanceOf(Date),
+    // festEndDate: PropTypes.instanceOf(Date),
```
Comment (patryem) [inline src/components/Fest.js:27]: delete if not in use

File: src/components/Fest.js [hunk 5]
```diff
@@ -49,29 +49,37 @@ export default class Fest extends Component {
       year: '2-digit'
     })
 
-    const festEndDateFormat = festEndDate.toLocaleDateString('en-GB', {
-      day: 'numeric',
-      month: '2-digit',
-      year: '2-digit'
-    })
+    let festEndDateFormat
+    if (festEndDate === '') {
```
Comment (patryem) [inline src/components/Fest.js:53]: you could check here for falsy statements

File: src/components/Fest.js [hunk 6]
```diff
@@ -49,29 +49,37 @@ export default class Fest extends Component {
...
+      console.log('ohne datum')
```
Comment (patryem) [inline src/components/Fest.js:54]: do not forget all the console log !!

File: src/components/Fest.js [hunk 7]
```diff
@@ -49,29 +49,37 @@ export default class Fest extends Component {
...
+        year: '2-digit'
+      })
+    }
 
     return (
       <StyledWrapper data-cy="festEl">
         <TimetableLink data-cy="festElLink" to={`/timetable/${festId}`}>
-          <time dateTime={festEndDateFormat}>
+          <time className="teal" dateTime={festEndDateFormat}>
             {festEndDateFormat === ''
```
Comment (patryem) [inline src/components/Fest.js:70]: falsy

**Review by patryem — COMMENTED** [review]
Minor things to improve your code. Good job btw !

File: src/components/App.js (hunk 2 above)
Comment (agnesbr) [reply src/components/App.js:34]: done

**Pull request merged**